.github/workflows/release.yml

```diff
@@ -98,7 +98,9 @@ jobs:
         with:
           context: .
           push: true
-          tags: ghcr.io/filecoin-saturn/l1-node:${{ env.SATURN_NETWORK }}
+          tags: |
+            ghcr.io/filecoin-saturn/l1-node:${{ env.SATURN_NETWORK }}
+            ghcr.io/filecoin-saturn/l1-node:${{ github.sha }}-${{ github.run_number}}
```

Comment:
```suggestion
            ghcr.io/filecoin-saturn/l1-node:${{ github.run_number}}_${{ github.sha }}
```

Reply:
This would match what's in code and is shared to the orch

Reply:
And would be better to have the short sha to exactly match: https://stackoverflow.com/questions/59810838/how-to-get-the-short-sha-for-the-github-workflow